[2024-10-02T17:26:34.986+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T17:26:35.012+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T17:26:35.026+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T17:26:35.027+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T17:26:35.285+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-19 00:00:00+00:00
[2024-10-02T17:26:35.293+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1651) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T17:26:35.294+0000] {standard_task_runner.py:63} INFO - Started process 1667 to run task
[2024-10-02T17:26:35.294+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-19T00:00:00+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpv_6q8sgq']
[2024-10-02T17:26:35.296+0000] {standard_task_runner.py:91} INFO - Job 166: Subtask transformar
[2024-10-02T17:26:35.354+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T17:26:35.447+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T00:00:00+00:00'
[2024-10-02T17:26:35.451+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T17:26:35.669+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T17:26:35.670+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/etl_pipeline.py", line 17, in transformar
    df_agrupada = df.groupby('ContinentExp')[['cases', 'deaths']].sum().reset_index()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py", line 8869, in groupby
    return DataFrameGroupBy(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/groupby.py", line 1278, in __init__
    grouper, exclusions, obj = get_grouper(
                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/groupby/grouper.py", line 1009, in get_grouper
    raise KeyError(gpr)
KeyError: 'ContinentExp'
[2024-10-02T17:26:35.683+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-19T00:00:00+00:00, execution_date=20231219T000000, start_date=20241002T172635, end_date=20241002T172635
[2024-10-02T17:26:35.696+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 166 for task transformar ('ContinentExp'; 1667)
[2024-10-02T17:26:35.749+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-02T17:26:35.771+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T17:26:35.776+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T18:58:36.523+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:58:36.557+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T18:58:36.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T18:58:36.854+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:58:36.872+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-19 00:00:00+00:00
[2024-10-02T18:58:36.881+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5931) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:58:36.883+0000] {standard_task_runner.py:63} INFO - Started process 5947 to run task
[2024-10-02T18:58:36.883+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-19T00:00:00+00:00', '--job-id', '1191', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp1tlfpvws']
[2024-10-02T18:58:36.885+0000] {standard_task_runner.py:91} INFO - Job 1191: Subtask transformar
[2024-10-02T18:58:36.945+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:58:37.051+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T00:00:00+00:00'
[2024-10-02T18:58:37.055+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:58:37.126+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:58:37.127+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:58:37.135+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-19T00:00:00+00:00, execution_date=20231219T000000, start_date=20241002T185836, end_date=20241002T185837
[2024-10-02T18:58:37.178+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:58:37.198+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:58:37.202+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:24:55.470+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:24:55.511+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T19:24:55.927+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2024-10-02T19:24:55.928+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:24:55.976+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformar> on 2023-12-19 00:00:00+00:00
[2024-10-02T19:24:55.992+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9303) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:24:55.995+0000] {standard_task_runner.py:63} INFO - Started process 9331 to run task
[2024-10-02T19:24:55.995+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'transformar', 'scheduled__2023-12-19T00:00:00+00:00', '--job-id', '2206', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp5z9v8oys']
[2024-10-02T19:24:56.000+0000] {standard_task_runner.py:91} INFO - Job 2206: Subtask transformar
[2024-10-02T19:24:56.130+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.transformar scheduled__2023-12-19T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:24:56.288+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='transformar' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T00:00:00+00:00'
[2024-10-02T19:24:56.295+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:24:56.514+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:24:56.516+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:24:56.527+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=transformar, run_id=scheduled__2023-12-19T00:00:00+00:00, execution_date=20231219T000000, start_date=20241002T192455, end_date=20241002T192456
[2024-10-02T19:24:56.586+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:24:56.615+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:24:56.620+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
